lib/active_support/cache/memcached_store.rb

```diff
@@ -55,40 +58,34 @@ def read_multi(*names)
         keys_to_names = Hash[names.map { |name| [normalize_key(name, options), name] }]
         values = {}
 
-        instrument(:read_multi, names, options) do
-          if raw_values = @data.get(keys_to_names.keys, false)
-            raw_values.each do |key, value|
-              entry = deserialize_entry(value)
-              values[keys_to_names[key]] = entry.value unless entry.expired?
+        handle_exceptions(return_value_on_error: {}) do
+          instrument(:read_multi, names, options) do
+            if raw_values = @data.get(keys_to_names.keys, false)
+              raw_values.each do |key, value|
+                entry = deserialize_entry(value)
+                values[keys_to_names[key]] = entry.value unless entry.expired?
+              end
             end
           end
+          values
         end
-        values
-      rescue Memcached::NotFound
```

Comment:
This was previously only rescuing NotFound, but now it's also rescueing ConnectionDataExists. Probably not a big deal. Is it?